extension/webpack.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
         InstructorDashboard: "./src/standalone_pages/InstructorDashboard.js",
         InstructorDashboardRenderVideoInfo: "./src/standalone_pages/InstructorDashboardRenderVideoInfo.js",
         MinervaMenu: "./src/popups/MinervaMenu.js",
+        MinervaMenuRenderInfo: "./src/popups/MinervaMenuRenderInfo.js",
```

Comment:
response?

Reply:
reponse

Reply:
The reason that files are specified here because these are considered to be entry points. Entry point files are files where webpack will essentially run DFS on all dependencies and glue everything together. Entry points are considered individual entities to be __included__ by the html. In your case, `MinervaMenuRenderInfo` is a dependency but not a root dependency.